bumble/att.py

```diff
@@ -778,22 +800,26 @@ def __init__(self, attribute_type, permissions, value=b''):
         else:
             self.value = value
 
-    def encode_value(self, value):
+    def encode_value(self, value: bytes) -> bytes:
         return value
 
-    def decode_value(self, value_bytes):
+    def decode_value(self, value_bytes: bytes) -> bytes:
         return value_bytes
 
-    def read_value(self, connection: Connection):
+    def read_value(self, connection: Connection | None) -> bytes:
```

Comment:
the `|` symbol for unions is only for Python >= 3.10, but we still support >= 3.8, so we should still use `Union[]` here.